benchmarks/README.md

```diff
@@ -88,3 +103,28 @@ gradlew -p benchmarks run --args ' MemoryStatsBenchmark -jvmArgs "-XX:+UnlockDia
 ```
 
 If you want `perf` to find the hot methods for you then do add `-prof:perfasm`.
+
+## Async Profiler
+
+Note: Instructions for Linux only, but it looks like async profiler supports mac as well.
+
+The async profiler is neat because it does not suffer from the safepoint
+bias problem. And because it makes pretty flame graphs!
+
+Let user processes read performance stuff:
+```
+sudo bash
+echo 0 > /proc/sys/kernel/kptr_restrict
+echo 1 > /proc/sys/kernel/perf_event_paranoid
+exit
+```
+
+Grab the async profiler from https://github.com/jvm-profiling-tools/async-profiler
+and run jmh with it in the LD_LIBRARY_PATH:
+```
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/Downloads/tmp/async-profiler-1.8.3-linux-x64/build/ \
```

Comment:
should we prefer the platform-independent `libPath` parameter to LD_LIBRARY_PATH? see https://github.com/openjdk/jmh/blob/master/jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java#L237

Reply:
I'm just used to LD_LIBRARY_PATH. I'll take what folks like.

Reply:
I only mention because us Apple folk would need to know, or else it would need to be alternately documented (or tripped on), that we use DYLD_LIBRARY_PATH to achieve the same thing

Reply:
I'd be quite happy if an apple folk could try with the libPath thing and see if it works for them. If so, I'm be happy to try on linux.

Reply:
```
./gradlew -p benchmarks/ run --args 'LongKeyedBucketOrdsBenchmark.multiBucket -prof "async:dir=/tmp/prof;output=flamegraph;libPath=/Users/rick.boyd/Downloads/async-profiler-1.8.3-macos-x64/build/libasyncProfiler.so"'
```

This worked for me, though I'll point out two things that I don't necessarily think need to be documented as caveats: 
a) the typical "this library was downloaded from the internet" security event occurs when you try to load it (fixed by going to settings and allowing it to run)
b) the containing directory did not work in this method, it required spelling out the full path to the lib

Reply:
libPath works for me too! I've pushed an update that includes it. I'm left a note about the "downloaded from the internet" warning but I don't know much about how to enable it so I was vague about it.